app/src/main/java/com/alexis/myanimecompanion/data/remote/models/SearchResult.kt

```diff
@@ -0,0 +1,18 @@
+package com.alexis.myanimecompanion.data.remote.models
+
+import com.alexis.myanimecompanion.domain.Anime
+
+data class SearchResult(
+    val data: List<Data> = listOf(),
+    // TODO use paging
+    val paging: Paging = Paging()
+)
+
+fun SearchResult.asListOfAnime(): List<Anime> {
+    val animeList = mutableListOf<Anime>()
+
+    for (data in this.data) {
```

Comment:
Can you rename the variable used to iterate as node or something different?

Reply:
I've renamed _this.data_ to _this.listOfData_. _data_ contains _node_.

Reply:
Great work!

Reply:
I would name it dataList instead